bin/zbackup

```diff
@@ -70,13 +70,13 @@ def get_backup_properties(zpool, options, tier = None):
         property_ids = "all"
     zfs_get = subprocess.Popen(["zfs", "get", "-H", "-r", "-t", "filesystem", property_ids, zpool], stdout = subprocess.PIPE)
     for line in zfs_get.stdout:
-        (name, property, value, source) = line.rstrip('\n').split('\t')
+        (name, property, value, source) = line.decode('ascii').rstrip('\n').split('\t')
```

Comment:
This does not work correctly here.  The correct solution is to:

1. Add `universal_newlines=True` to `subprocess.Popen`.
2. Skip the `decode()` call.